src/App.js

```diff
@@ -1,42 +1,134 @@
 import React from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
-import TodoListItem from './TodoListItem';
+// import TodoListItem from './TodoListItem';
+// import { json } from 'express';
```

Comment:
Please delete commented-out lines since they're not being used. If you want to make sure you have them saved for later, we can make sure they're checked in as part of a commit before you remove them. That way they'll be in the permanent history of the git repository if we ever want to come back and refer to them =D

Reply:
Ok